cli-toolchain/quickstart.md

```diff
@@ -117,6 +117,50 @@ Do you want to use the default authentication and security configuration?
   I want to learn more.
 ```
 
+#### Adding a Lambda Trigger
+
+There are two ways to setup Lambda Triggers for your Cognito User Pool.
```

Comment:
Let's add some detail text:

"Lambda triggers are useful for adding functionality during certain lifecycles of the user registration and sign-in process of your application. Amplify ships common triggers which you can enable and modify (if necessary) through a few simple questions. Alternatively, you can build your own auth challenges manually."